File: patterns/javascript-patterns/pattern118.js
// Pattern 118: ...

// ******
// ******
//   ****
//   ****
//     **
//     **

let n = 6;

for (let i = n; i > 0; i--) {
    let k = (i % 2 !== 0) ? i + 1 : i;

    let line = '';
    for (let x = k; x < n; x++) {
        line += ' ';
    }
    for (let j = 0; j < k; j++) {
        line += '*';
    }

    console.log(line);
}

File: patterns/python-patterns/pattern118.py
# Pattern 118: ...

# ******
# ******
#   ****
#   ****
#     **
#     **

n = 6

for i in range(n, 0, -1):
    k = i + 1 if i % 2 != 0 else i
    print(" " * (n - k) + "*" * k)
